docs/snippets/vpu/custom_op.cpp

```diff
@@ -1,11 +1,10 @@
 #include "openvino/runtime/core.hpp"
-#include "ie_plugin_config.hpp"
 
 int main() {
 //! [part0]
 ov::Core core;
 // Load Myriad Extensions
-core.set_property("MYRIAD", {{CONFIG_KEY(CONFIG_FILE), "<path_to_the_xml_file>"}});
+core.set_property("MYRIAD", {{"CONFIG_FILE", "<path_to_the_xml_file>"}});
```

Comment:
I suppose we need to remove VPU custom ops, because MYRIAD is removed from OpenVINO more than 1 year ago.
Looks like we just forgot to drop some files

Reply:
Does change `MYRIAD `to `NPU `make sense?

Reply:
no. They are completely different devices.